app/src/main/java/com/nitrr/ecell/esummit/ecellapp/misc/PrefUtils.java

```diff
@@ -22,4 +24,49 @@ public String getAccessToken() {
         SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
         return prefs.getString("access_token", null);
     }
+
+    public void clearPrefs() {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
+        SharedPreferences.Editor editor = prefs.edit();
+        editor.clear();
+        editor.apply();
+    }
+
+    public void setIsLoggedIn(boolean isLoggedIn) {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
+        SharedPreferences.Editor editor = prefs.edit();
+        editor.putBoolean("isLoggedIn", isLoggedIn);
+        editor.apply();
+    }
+
+    public boolean getIsLoggedIn() {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
+        return prefs.getBoolean("isLoggedIn", false);
+    }
+
+    public void setIsFBLoggedIn(boolean isFBLoggedIn) {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
+        SharedPreferences.Editor editor = prefs.edit();
+        editor.putBoolean("isFBLoggedIn", isFBLoggedIn);
+        editor.apply();
+    }
+
+    public boolean getIsFBLoggedIn() {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
+        return prefs.getBoolean("isFBLoggedIn", false);
+    }
+
+    public void setIsGLoggedIn(boolean isGLoggedIn) {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
+        SharedPreferences.Editor editor = prefs.edit();
+        editor.putBoolean("isFBLoggedIn", isGLoggedIn);
+        editor.apply();
+    }
+
+    public boolean getIsGLoggedIn() {
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(activity);
+        return prefs.getBoolean("isGLoggedIn", false);
+    }
+
```

Comment:
Store every login in a single Java object and save that object in SharedPrefs. Get rid of un-nessasary function and function calls.